src/DW1000Ng.cpp

```diff
@@ -138,19 +138,79 @@ namespace DW1000Ng {
 					headerLen += 2;
 				}
 			}
+			
 			SPIporting::writeToSPI(_ss, headerLen, header, data_size, data);
 		}
 
-		void _writeToRegister(byte cmd, uint16_t offset, uint32_t data, uint16_t data_size) { 
+		/*
+		* Write Value in Hex or Int format to the DW1000. Single Value can be written to registers via sub-addressing.
+		* @param cmd
+		* 		The register address (see Chapter 7 in the DW1000 user manual).
+		* @param offset
+		*		The offset to select register sub-parts for writing, or 0x00 to disable
+		* 		sub-adressing.
+		* @param data
+		*		The data Value to be written.
+		* @param data_size
+		*		The number of bytes to be written
+		*/
+		void _writeValueToRegister(byte cmd, uint16_t offset, uint32_t data, uint16_t data_size) { 
 			byte dataBytes[data_size];
+			// Converts input Value in Byte and writes it inside the Array
 			DW1000NgUtils::writeValueToBytes(dataBytes, data, data_size);
+			// Reuses the core function to write the register
```

Comment:
This type of comment is pointless as it explains what the functions does, it is best suited in the header of that function instead.

Reply:
Yes, you are right!